deployment/Tiltfile

```diff
@@ -9,10 +9,25 @@ k8s_resource(
 )
 
 docker_build(
- 'spryker-image',
+ 'spryker-image-base',
  '..',
  target='spryker',
  dockerfile='Dockerfile',
+)
+
+docker_build(
+ 'spryker-image',
+ '..',
+ dockerfile_contents='FROM spryker-image-base',
+ live_update=[
+   sync('..', '/data'),
+ ],
+)
+
+docker_build(
+ 'spryker-image-tasks',
+ '..',
+ dockerfile_contents='FROM spryker-image-base',
  live_update=[
    sync('..', '/data'),
```

Comment:
If this image is used by jobs only, you could possibly omit live updates here as well since they don't really apply.